python/cudf/cudf/tests/test_fill.py

```diff
@@ -0,0 +1,40 @@
+import pytest
+from pandas.util.testing import assert_series_equal
+
+import cudf
+
+
+def cpu_fill(data, fill_value, begin, end):
+
+    if end == -1:
+        end = len(data)
+
+    fill_size = end - begin
+
+    a = list(data[i] for i in range(begin))
+    b = list(fill_value for i in range(fill_size))
+    c = list(data[i + begin + fill_size] for i in range(len(data) - end))
+
+    return a + b + c
```

Comment:
Instead of this function can likely just use `pd.Series(...)[start:end] = fill_value`